pkg/printers/internalversion/describe.go

```diff
@@ -837,6 +839,31 @@ func printConfigMapVolumeSource(configMap *api.ConfigMapVolumeSource, w PrefixWr
 		configMap.Name, optional)
 }
 
+func printProjectedVolumeSource(projected *api.ProjectedVolumeSource, w PrefixWriter) {
+	infos := []string{}
+	for _, source := range projected.Sources {
+		if source.Secret != nil {
+			infos = append(infos, fmt.Sprintf("SecretName:\t%v\n"+
+				"    SecretOptionalName:\t%v\n",
+				source.Secret.Name, source.Secret.Optional))
+		} else if source.DownwardAPI != nil {
+			infos = append(infos, "DownwardAPI\n")
+		} else if source.ConfigMap != nil {
+			infos = append(infos, fmt.Sprintf("ConfigMapName:\t%v\n"+
+				"    ConfigMapOptional:\t%v\n",
+				source.ConfigMap.Name, source.ConfigMap.Optional))
+		} else if source.ServiceAccountToken != nil {
+			infos = append(infos, fmt.Sprintf("TokenExpirationSeconds:\t%v\n",
+				source.ServiceAccountToken.ExpirationSeconds))
+		}
+	}
+
+	w.Write(LEVEL_2, "Type:\tProjected (a volume that contains injected data)\n")
```

Comment:
`s/a volume that contains injected data/a volume that contains injected data from multiple sources/` ?